application/src/test/java/org/opentripplanner/apis/gtfs/service/ApiTransitServiceTest.java

```diff
@@ -21,6 +21,14 @@
 import org.opentripplanner.updater.trip.TripInput;
 import org.opentripplanner.updater.trip.TripUpdateBuilder;
 
+/**
+ * This test uses the real-time test environment to establish the condition under test.
+ * This pulls in a bunch of dependencies, namely on GTFS-RT classes, that go a bit beyond the scope
+ * of this bug, however, the other option is to write to the timetable snapshot/repository directly.
```

Comment:
bug?

Reply:
I wanted to express that the bug doesn't have anything to do with GTFS-RT but maybe it was misleading. I will change it.